scripts/image

```diff
@@ -68,6 +68,14 @@ if [ -n "$CUSTOM_IMAGE_NAME" ]; then
    IMAGE_NAME="$CUSTOM_IMAGE_NAME" 
 fi
 
+if [ -n "$CUSTOM_VERSION" ]; then
+  LIBREELEC_VERSION="$CUSTOM_VERSION"
```

Comment:
What about the places where `LIBREELEC_VERSION` is already used, https://github.com/plexinc/LibreELEC.tv/blob/upstream-custom/scripts/image#L57 - shouldn't you set your `CUSTOM_VERSION` before line 57, otherwise the image may be named differently to the version referenced elsewhere within the image (ie. `/etc/os-release`).

No doubt you're setting a `CUSTOM_IMAGE` too, but if the user is only setting a `CUSTOM_VERSION` then things could become a little disconnected, with the image being named using the standard computed version, which is also embedded in `/'etc/release` (as `TARGET_VERSION`), however the `CUSTOM_VERSION` will then be embedded in `/etc/os-release`, which won't match `TARGET_VERSION`.

tl;dr: I think `CUSTOM_VERSION` processing should occur before line 57.